aidants_connect/settings.py

```diff
@@ -421,9 +421,17 @@ def getenv_bool(key: str, default: Optional[bool] = None) -> bool:
     "'sha256-oOHki3o/lOkQD0J+jC75068TFqQoV40dYK6wrkIXI1c='",  # statistiques.html
     "https://cdnjs.cloudflare.com/ajax/libs/chartjs-plugin-datalabels/2.0.0/chartjs-plugin-datalabels.min.js",
     "'sha256-CO4GFu3p1QNoCvjdyc+zNsVh77XOc5H2OcZYFb8YUPA='",  # home_page.html
+    "https://code.jquery.com/jquery-3.6.1.js",
+    "https://code.jquery.com/ui/1.13.1/jquery-ui.js",
+    "'sha256-MTmHTCNXu/Je/qmoKw/rxgZTtLA1QVKkrkv7XM0V1SY='",  # authorize.html
+)
+
+CSP_STYLE_SRC = (
+    "'self'",
+    "data",
+    "https://code.jquery.com/ui/1.13.2/themes/base/jquery-ui.css",
```

Comment:
```suggestion
CSP_STYLE_SRC = (
    "'self'",
    "https://code.jquery.com/ui/1.13.2/themes/base/jquery-ui.css",
```

C'est `"data:"` et faut le mettre dans `CSP_IMG_SRC`. Ça permet d'autoriser les images décrite en ligne en base64. Mais finalement pas besoin, j'ai embarqué cette modif dans #758.